src/App.css

```diff
@@ -115,9 +114,31 @@ nav ul.navbar-nav li:last-child>div {
   position: sticky;
   right: -30px;
   bottom: 10px;
-  white-space: nowrap
+  white-space: nowrap;
 }
 
 tr {
   content-visibility: auto;
-}
\ No newline at end of file
+}
+
+.floating-header {
+  position: fixed; /* Keeps the button in the same position on the screen */
```

Comment:
Hi, I was a bit curious about the responsive effect, thus have a try. I don't think using `fixed` is good here as you may check how it looks like in the video. 
The first half is the `development` branch while when I move my mouse around the **element** in the right panel, it's this PR and the header is almost like a nav bar. I guess it's not part of this PR? If I'm wrong, please correct me.

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/57503876/1dcd6b2e-82bb-489a-bc41-7461f70f7752